### DISCUSSION TITLE
Unload a RAM font and How to make own font

### DISCUSSION THREAD

**Discussion by valdus2** [arendst/Tasmota#21054] [Q&A] [answered]
1) With command Ffilename I can load and use a ram font. Now is there a command to unload it from ram, so I can load another one? 

2) I use the sans font from folder BinFonts, but there is no utf-8 support, so no international characters. 
How can I make may own special binary font version?
Are there any instructions or maybe an editor so I can convert a font type which includes for example Greek characters?

**Comment by gemu2015** [comment, marked as answer]
1. there is no unload, just load another, it replaces the last one.
2. there is dedicated font editor and font converter included in script editor:
https://www.dropbox.com/sh/0us18ohui4c3k82/AACcVmpZ4AfpdrWE_MPFGmbma?dl=0

**Reply by valdus2** [reply]
Ah, OK. Thanks for clarifying this.
